keyboards/aplusx_pulsar/xboard_qs/keymaps/via/keymap.c

```diff
@@ -0,0 +1,69 @@
+/* Copyright 2024 Aplusx Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+#include "xboard_qs.h"
```

Comment:
This isn't needed, as the build process generates `QMK_KEYBOARD_H` to do this.

```suggestion
```

Reply:
I got it. thank you